tests/webview/ui/test_content.py

```diff
@@ -766,3 +766,36 @@ def test_chapter_review_version_matches_book_version(webview_base_url, selenium,
 
     # THEN the chapter review version matches the book version
     assert content.page_version == content.book_version
+
+
+@markers.webview
+@markers.test_case('C195071')
+@markers.nondestructive
+@markers.parametrize('id', ['yWshIYVW@1'])
+def test_buggy_msg_not_show_in_left_panel(webview_base_url, id, selenium):
+    # GIVEN the webview base url, a chapter page id, and the Selenium driver
+
+    # WHEN we visit that page of the chapter
+    content = Content(selenium, webview_base_url, id=id).open()
+
+    # THEN correct book info is listed in the left panel
+    if content.is_left_panel_title_displayed:
+        assert content.is_go_to_book_link_present
+
+
+@markers.webview
+@markers.test_case('C195061')
+@markers.nondestructive
+@markers.parametrize('id', ['BWYBGK7C@2'])
+def test_book_title_not_limited(webview_base_url, id, selenium):
+    # GIVEN the webview base url, a chapter page id, and the Selenium driver
+
+    # WHEN we visit that page of the chapter
+    content = Content(selenium, webview_base_url, id=id).open()
+
+    # THEN get the title text to check if it matches what it suppose to be
+    actual_title = content.get_left_nav_book_title
```

Comment:
the `get_left_nav_book_title` is ok but when we get text we simply reference the text attribute. Change this to work in this fashion:
`book_title = content.left_nav_book_title.text`